models/public/midasnet/midasnet.md

```diff
@@ -0,0 +1,41 @@
+# midasnet
+
+## Use Case and High-Level Description
+
+MidasNet is a model for monocular depth estimation trained by mixing several datasets;
+as described in the following paper:
+"Towards Robust Monocular Depth Estimation: Mixing Datasets for Zero-Shot Cross-Dataset Transfer"
+<https://arxiv.org/pdf/1907.01341>
+
+The model input is a blob that consists of a single image of "1x3x384x384" in `BGR` order.
+
+The model output is an inverse depth map that is defined up to an unknown scale factor.
+
+## Example
+
+See [here](https://github.com/intel-isl/MiDaS)
+
+## Input
```

Comment:
Usually, we describe inputs and outputs for both original and converted in IR models, even if they are same. Once again, see [as example](https://github.com/opencv/open_model_zoo/blob/develop/models/public/alexnet/alexnet.md).

Reply:
done